Formula/elektra.rb

```diff
@@ -0,0 +1,42 @@
+class Elektra < Formula
+  desc "Configuration Framework"
+  homepage "https://libelektra.org"
+  url "https://www.libelektra.org/ftp/elektra/releases/elektra-0.8.21.tar.gz"
+  sha256 "51892570f18d1667d0da4d0908a091e41b41c20db9835765677109a3d150cd26"
+  head "https://github.com/ElektraInitiative/libelektra.git"
+
+  option "with-qt", "Build GUI frontend"
+
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "qt" => :optional
+  depends_on "discount" if build.with? "qt"
+
+  def install
+    tools = ["kdb"]
+    tools << "qt-gui" if build.with? "qt"
+
+    cmake_args = std_cmake_args + %W[
```

Comment:
Let's use `args` rather than `cmake_args`. The latter seems to have crept into a few places but the former is still overwhelmingly the standard syntax.